ext/lmdb-safe/lmdb-typed.hh

```diff
@@ -234,7 +232,7 @@ class TypedDBI
 
 private:
   template <uint8_t N>
-  auto openDB(string_view& name)
+  inline auto openDB(string_view& name)
```

Comment:
I thought `inline` was implicit for the definition case (body following declaration), see https://en.cppreference.com/w/cpp/language/inline, 2nd parapraph.

Reply:
Well, I'd rather keep it still:
- It doesn't hurt.
- It conveys intention (and since I'm replacing a macro here...).
- Explicit is better than implicit.
- More generally, if code is moved somewhere, it can be copy-pasted without potentially regressing (probably not the case here but in general this stands). e.g. `unless it is attached to a named module (since C++20)`
- The compiler does what the compiler wants anyway so why do we care :-D `Since inline substitution is unobservable in the standard semantics, compilers are free to use inline substitution for any function that's not marked inline, and are free to generate function calls to any function marked inline. Those optimization choices do not change the rules regarding multiple definitions and shared statics listed above.`

Reply:
OK, agreed either way.